formula-android/src/main/java/com/instacart/formula/android/FormulaFragment.kt

```diff
@@ -130,22 +139,8 @@ class FormulaFragment : Fragment(), BaseFormulaFragment<Any> {
         val output = output ?: return
         val view = featureView ?: return
 
-        val fragmentId = getFormulaFragmentId()
-        val environment = FormulaFragmentDelegate.fragmentEnvironment()
-        val fragmentDelegate = environment.fragmentDelegate
-
         try {
-            fragmentDelegate.setOutput(fragmentId, output, view.setOutput)
-
-            if (firstRender) {
-                val end = SystemClock.uptimeMillis()
-
-                firstRender = false
-                fragmentDelegate.onFirstModelRendered(
```

Comment:
We don't need this anymore because we can use `onNewInstance` with `setOutput` to fire the first model render